Content.Server/PneumaticCannon/PneumaticCannonComponent.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Content.Shared.Sound;
+using Content.Shared.Tool;
+using Content.Shared.Verbs;
+using Robust.Shared.Analyzers;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.PneumaticCannon
+{
+    [RegisterComponent, Friend(typeof(PneumaticCannonSystem))]
+    public class PneumaticCannonComponent : Component
+    {
+        public override string Name { get; } = "PneumaticCannon";
+
+        [ViewVariables]
+        public ContainerSlot GasTankSlot = default!;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public PneumaticCannonPower Power = PneumaticCannonPower.Low;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        public PneumaticCannonFireMode Mode = PneumaticCannonFireMode.Single;
+
+        public ToolQuality ModifyPower = ToolQuality.Screwing;
+        public ToolQuality ModifyMode = ToolQuality.Anchoring;
```

Comment:
Is this really something that needs to be specified as a field here? It's literally never changed so it feels like just inlining the values makes much more sense. If this is supposed to be modified it should have VV/DataField instead, but I doubt that.